recipes/pyuoi/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyuoi" %}
+{% set version = "0.0.61" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4085faf3c6dd2add38fb199350c50fcccbe6eafe78c07ec25844ca4d773d7095
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<35]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
Do you need numpy here? You probably only need it if you’re using the C API and, if you are, check the conda forge docs for how to pin it at runtime.

Reply:
We build against numpy and I've updated things to what I believe is correct from here
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy